test_cases/README.md

```diff
@@ -1,31 +1,35 @@
 ## Regression tests for typeshed
 
-This directory contains regression tests for the stubs found elsewhere in the
-typeshed repo.
+This directory contains code samples that act as a regression test for the
+standard-library stubs found elsewhere in the typeshed repo.
```

Comment:
```suggestion
standard library stubs found elsewhere in the typeshed repo.
```

Reply:
Not sure about this one. "Standard library" is a compound noun here; without the hyphen, you could misread it as "library stubs that are standard" rather than "stubs for the standard library".

Reply:
Well, it's called "The Python Standard Library" and it's consistently spelled "standard library" in the Python docs.

Reply:
Meh, we can agree to disagree; I'll drop the hyphen